<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>255</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QPushButton" name="btNum0">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>snack_picture/number/number_0.png</normaloff>snack_picture/number/number_0.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btNum4">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>snack_picture/number/number_4.png</normaloff>snack_picture/number/number_4.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btNum7">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>snack_picture/number/number_7.png</normaloff>snack_picture/number/number_7.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btNum5">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>snack_picture/number/number_5.png</normaloff>snack_picture/number/number_5.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btNum6">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>snack_picture/number/number_6.png</normaloff>snack_picture/number/number_6.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btNum3">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>snack_picture/number/number_3.png</normaloff>snack_picture/number/number_3.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btNum2">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>snack_picture/number/number_2.png</normaloff>snack_picture/number/number_2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btNum8">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>snack_picture/number/number_8.png</normaloff>snack_picture/number/number_8.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btNum9">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>snack_picture/number/number_9.png</normaloff>snack_picture/number/number_9.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btNum1">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>snack_picture/number/number_1.png</normaloff>snack_picture/number/number_1.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>60</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="tbPwShow">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbPw">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btPwCheck">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btPwCancel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>지우기</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbGuide">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>비밀번호를 입력해 주세요</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
